### PR TITLE
Update README.md for Collectd

### PR DESCRIPTION
**Pull request by nicgrayson**
- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

### CODE REVIEW

**Comment by corylanou** [comment]
Looks good.  Can you add a changelog entry and confirm you have signed the CLA.  Thanks!

**Comment by nicgrayson** [comment]
@corylanou squashed, changelog'd, and signed

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 - [#5479](https://github.com/influxdata/influxdb/issues/5479): Bringing up a node as a meta only node causes panic
 - [#5504](https://github.com/influxdata/influxdb/issues/5475): create retention policy on unexistant DB crash InfluxDB
 - [#5505](https://github.com/influxdata/influxdb/issues/5505): Clear authCache in meta.Client when password changes.
-
```
Comment (jwilder) [inline CHANGELOG.md]: This needs to go under the `v0.11.0` section now.  `v.0.10.0` has already been released.

**Comment by nicgrayson** [comment]
@jwilder sorry about that. I rebased and fixed.

**Comment by corylanou** [comment]
+1

**Comment by e-dard** [comment]
@nicgrayson needs another rebase and I can get this merged.

**Comment by nicgrayson** [comment]
@e-dard rebased

**Pull request merged**